tests/src/test/scala/cats/tests/NonEmptyListSuite.scala

```diff
@@ -84,86 +87,86 @@ class NonEmptyListSuite extends NonEmptyCollectionSuite[List, NonEmptyList, NonE
 
   test("Show is not empty and is formatted as expected") {
     forAll { (nel: NonEmptyList[Int]) =>
-      nel.show.nonEmpty should ===(true)
-      nel.show.startsWith("NonEmptyList(") should ===(true)
-      nel.show should ===(implicitly[Show[NonEmptyList[Int]]].show(nel))
-      nel.show.contains(nel.head.show) should ===(true)
+      assert(nel.show.nonEmpty === (true))
```

Comment:
those asserting `=== true` could be
```suggestion
      assert(nel.show.nonEmpty)
```